src/sentry/search/eap/spans/formulas.py

```diff
@@ -18,6 +18,15 @@
 from sentry.search.eap.constants import RESPONSE_CODE_MAP
 from sentry.search.eap.utils import literal_validator
 
+TOTAL_SPAN_COUNT = Column(
+    aggregation=AttributeAggregation(
+        aggregate=Function.FUNCTION_COUNT,
+        key=AttributeKey(type=AttributeKey.TYPE_DOUBLE, name="sentry.exclusive_time_ms"),
```

Comment:
Double checking, do we need the key at all?

Reply:
Yeah, we get yelled at without the key
<img width="589" alt="image" src="https://github.com/user-attachments/assets/5f644aad-7d48-453c-becf-d813b864b865" />

Reply:
perfect